### PR TITLE
login index is created

### PR DESCRIPTION
**Pull request by Laziz222**

### CODE REVIEW

**Comment by Laziz222** [comment]
i dont have to pull request,right?

File: login/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,946 @@
+
```
Comment (azNodirbek) [inline login/css/styles.css]: In this .css file you have made a lot of spaces which resulted with huge unused lines, reduce it

File: login/index.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	<div class="row">
+		<div class="col-xs-10 col-xs-offset-1 col-sm-8 col-sm-offset-2 col-md-4 col-md-offset-4">
+			<div class="login-panel panel panel-default">
+				<div class="panel-heading">Log in as Admin</div>
+				<div class="panel-body">
+					<form role="form" method="post">
+						<fieldset>
+							<div class="form-group">
+								<input class="form-control" placeholder="E-mail" name="email" type="email" autofocus="" required>
+							</div>
```
Comment (azNodirbek) [inline login/index.php:30]: In this file you have syntax error, solve it

File: login/index.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+								</form>
+							</div>
+
+					</div>
+				</div><!-- /.col-->
+			</div><!-- /.row -->	
+		</div>
+	</div>	
+</body>
+</html>
```
Comment (azNodirbek) [inline login/index.php:39]: make here one free line under you code

File: login/css/styles.css [hunk 4]
```diff
@@ -0,0 +1,778 @@
...
+	display: none;
+}
+.icon-grid pre {
+	margin: 10px 10px -10px 10px;
+	border-radius: 0;
+	font-size: 10px;
+	border-color: #ddd;
+	height: 65px;
+	overflow: scroll;
+}
```
Comment (azNodirbek) [inline login/css/styles.css]: here also, one free line

File: login/css/styles.css (hunk 1 above)
Comment (Laziz222) [reply login/css/styles.css]: thank you. i m in the proccess)

File: login/index.php (hunk 2 above)
Comment (Laziz222) [reply login/index.php:30]: ok

File: login/index.php (hunk 3 above)
Comment (Laziz222) [reply login/index.php:39]: got it

File: login/css/styles.css (hunk 4 above)
Comment (Laziz222) [reply login/css/styles.css]: ok

**Comment by Laziz222** [comment]
given commetns are corrected

**Pull request merged**